ross/rotor_assembly.py

```diff
@@ -1744,7 +1791,7 @@ def plot_ucs(self, stiffness_range=None, num=20, ax=None, output_html=False):
                 self.shaft_elements, self.disk_elements, bearings, n_eigen=16
             )
             modal = rotor.run_modal(speed=0)
-            rotor_wn[:, i] = modal.wn[:8:2]
+            rotor_wn[:, i] = modal.wn[: self.number_dof * 2 : self.number_dof / 2]
```

Comment:
![image](https://user-images.githubusercontent.com/45969994/80371511-66bf2280-8868-11ea-97c8-677efdf2f1d2.png)

Here, we also have an error message. It says list index must be an interger of None. `self.number_dof` is a int by definition, however, when you divide it by 2, it becomes a float. So, you can solve it doing:

`rotor_wn[:, i] = modal.wn[: self.number_dof * 2 : int(self.number_dof / 2)]`

or similar

Reply:
Oh, we have had this problem with the 'number_dofs' becoming float everywhere... thanks for noticing this one! I will correct it right away.

Reply:
You're welcome!

Ahh, instead of adding a `int()` command, like a I said, do:
`rotor_wn[:, i] = modal.wn[: self.number_dof * 2 : self.number_dof // 2]`
it will force an integer output.

Reply:
Done